newlib/libc/sys/vita/error.c

```diff
@@ -45,174 +45,174 @@ int __vita_make_sce_errno(int posix_errno)
 
 int __vita_scenet_errno_to_errno(int sce_errno)
 {
-	switch (sce_errno)
+	switch (sce_errno & SCE_ERRNO_MASK)
 	{
-		case SCE_NET_ERROR_EPERM:
+		case SCE_NET_EPERM:
```

Comment:
could you explain why did you change this?

Reply:
It's explained in pullrequest description

Reply:
oh... got it... can you add the information into commit message?

Reply:
Done